core-bundle/src/EventListener/DataContainer/CustomTemplateOptionsListener.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\EventListener\DataContainer;
+
+use Contao\Controller;
+use Contao\CoreBundle\Framework\ContaoFramework;
+use Contao\DataContainer;
+use Symfony\Component\HttpFoundation\RequestStack;
+
+class CustomTemplateOptionsListener
```

Comment:
Could we maybe just go with `TemplateOptionsListener`? Would also shorten the service name 😬